deploy/sample-backup-configs/sample_config_runtime.ini

```diff
@@ -57,6 +57,10 @@ suffixtag=
 # In general, this is not required to produce deterministic simulations on
 # target machines running linux. Enable if you observe simulation non-determinism.
 zerooutdram=no
+# For designs built with synthesized assertions, enabling (=no) prints
```

Comment:
Replacing "enabling (=no)" with "setting disable_synth_asserts=no" and later "Setting disable_synth_asserts=yes" makes this read more intuitively.